portality/settings.py

```diff
@@ -484,7 +484,8 @@
 #                "index": False,
                 "store": True
             }
-        }
+        },
+        "copy_to": ["all_meta"]
```

Comment:
Doing this on the type means that *all* objects which have unicode types now have a `copy_to` for `all_meta`, and I don't think we want that.  For example, this rule now appears in BackgroundJob records too.

We should look for an approach that adds this only when necessary, probably a type-specific override to these defaults for Journals and Applications